.github/workflows/cmake.yml

```diff
@@ -219,7 +219,7 @@ jobs:
       run: |
         mkdir -p Build
         cd Build
-        cmake -GNinja -DBUILD_LAGOM=ON -DINCLUDE_WASM_SPEC_TESTS=ON -DWASM_SPEC_TEST_SKIP_FORMATTING=ON -DENABLE_UNDEFINED_SANITIZER=ON -DENABLE_ADDRESS_SANITIZER=ON -DENABLE_PCI_IDS_DOWNLOAD=OFF -DCMAKE_C_COMPILER=gcc-10 -DCMAKE_CXX_COMPILER=g++-10 ..
+        cmake -GNinja -DBUILD_LAGOM=ON -DINCLUDE_WASM_SPEC_TESTS=ON -DWASM_SPEC_TEST_SKIP_FORMATTING=ON -DENABLE_ADDRESS_SANITIZER=ON -DENABLE_PCI_IDS_DOWNLOAD=OFF -DCMAKE_C_COMPILER=gcc-10 -DCMAKE_CXX_COMPILER=g++-10 ..
```

Comment:
ASAN is much heavier on compile times than UBSAN. if we're removing UBSAN from even Lagom builds to reduce build times, it would make sense to remove ASAN as well.

Given that, it might make more sense to call the custom workflow sanitizer.yaml instead of ubsan.yaml? Then it would be a natural place to put KCOV or syzkaller jobs in the future.

Reply:
sounds good to me, ill rename the file and move ASAN to there as well